nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -1,6 +1,6 @@
 # Release 22.11 (“Raccoon”, 2022.11/??) {#sec-release-22.11}
```

Comment:
Actually, we should probably change this too :)

```suggestion
# Release 22.11 (“Raccoon”, 2022.11/30) {#sec-release-22.11}
```

Reply:
Oh, I see this will be done in https://github.com/NixOS/nixpkgs/pull/203735, sorry for the noise; ignore me x)